treeherder/perf/models.py

```diff
@@ -227,9 +229,47 @@ def _compute_ideal_chunk_size(self, max_timestamp, max_chunk_size):
         return len(older_ids) or max_chunk_size
 
 
+class Counter(models.Model):
```

Comment:
`null` parameter defaults to `False` [in Django](https://docs.djangoproject.com/en/3.0/ref/models/fields/#null). We should avoid being explicit about this, as this is also a general project convention.